README.md

```diff
@@ -28,6 +28,8 @@
   </a>
 </p>
 
+> _*Warning*_: Currently this project has an issue being tracked [here](https://github.com/ngrok/kubernetes-ingress-controller/issues/208) which can cause the controller to provide ingress to a service even if it failed to configure an authentication module like Oauth on a particular route due to a configuration or intermittent error. Until this issue is resolved, it's not recommended to use this with security sensitive applications.
```

Comment:
This makes it sound like it can only happen due to errors ('configuration or intermittent error').

This also happens racily in normal operation of the ingress controller with no errors anywhere. I filed a separate issue for that since I think that's also worth noting here: https://github.com/ngrok/kubernetes-ingress-controller/issues/219

Reply:
Updated to include this issue as well